vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1174,6 +1176,80 @@ def fused_experts(hidden_states: torch.Tensor,
             block_shape=block_shape)
 
 
+# This is a new triton implementation of the fused_experts function
+# Note: this only supports outplace and automatically renormalizes
+# TODO: add support for more arguments (see fused_experts)
```

Comment:
to avoid having boilerplate assertions for unsupported arguments, I just removed them